Sources/Controllers/Cells/OACollectionViewCellState.h

```diff
@@ -0,0 +1,15 @@
+//
+//  OACollectionViewCellState.h
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 20.04.2021.
+//  Copyright © 2021 OsmAnd. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface OACollectionViewCellState : NSObject
+
+@property (nonatomic) NSMutableDictionary<NSString *, NSValue *> *values;
```

Comment:
Rather than exposing the underlying collection, please add appropriate methods to get and set offset for a cell